history/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# NGINX base image
+FROM nginx:1.18.0-alpine
```

Comment:
Why nginx is needed? To just serve static files without a server?

Reply:
This is a server where vueApp is deployed, this is similar to tomcat and will integrate through gateway and so on. I think this is a super simple frontend app

Reply:
OK, where to get data? I.e. for displaying the history? Kafka?

Reply:
Current Service -> Gateway -> History Data Service